test/Altinn.AccessManagement.Tests/Data/Resources/resourceList.json

```diff
@@ -167,14 +173,18 @@
     },
     "homepage": "https://www.jksaudi.no/",
     "status": "Active",
+    "validFrom": "2022-09-20T06:46:07Z",
```

Comment:
These properties was also removed deliberately as they no longer are valid properties on the ServiceResource model

Reply:
removed these